### DISCUSSION TITLE
Tailwind CSS support ?

### DISCUSSION THREAD

**Discussion by chrisvidal** [OFFLINE-GmbH/oc-boxes-support#165] [Help] [answered]
I'm interested into experimenting Boxe on a new plugin.

Im using Tailwind CSS on all projects, so Io am wondering how does this integrate with if this is possible ?

thank you in advance

**Comment by tobias-kuendig** [comment]
There's nothing special to it. You integrate Tailwind in your Theme as usual, then just make sure to add the boxes directory to your Tailwind config's `content` key and you should be good to go.

**Reply by tobias-kuendig** [reply, marked as answer]
For exaple:

```js
// in themes/your-theme/tailwind.config.js

const basepath = __dirname + '/../../'

/** @type {import('tailwindcss').Config} */
module.exports = {
    content: [
        `${basepath}/themes/**/*.htm`,
        `!${basepath}/themes/**/assets`,
        `${basepath}/plugins/**/*.htm`,
        `${basepath}/storage/cms/twig/**/*.php`,
    ],
    ...
}
```

**Reply by chrisvidal** [reply]
I see .. so there is nothing to do on Boxes plugin backend side of things ?
I was thinking about how Boxes will render the partials if it does not have the compiled Tailwind CSS from the theme ?
